### ISSUE TITLE
Button grid odd right corners

### ISSUE THREAD

**Issue by nbuso** [ebiwd/EBI-Framework#95]
When you place buttons in a grid a right border is added, but it's not extending correctly to the top and bottom. Maybe it's better a margin more than a border?

![image](https://user-images.githubusercontent.com/7534952/27683332-76bdccb0-5cbd-11e7-9eb8-cc753141230c.png)

**Comment by khawkins98** [comment]
Good catch. I could swear this wasn't happening before, but it seems it was. 

Unfortunately a `margin-right` breaks the grid spacing:
![image](https://user-images.githubusercontent.com/928100/27760321-8e9cad94-5e3c-11e7-9aa8-6751e0627742.png)

The triangles are caused by `.button` having a 1px transparent border, one workaround is to add this css:
```css
body .button-grid a.button.columns {
  border: 0;
  border-right: 1px solid #fff;
}
```

Which gives you the desired:
![image](https://user-images.githubusercontent.com/928100/27760366-fcc4ad2a-5e3d-11e7-98b0-16ec2002932b.png)

I'm hesitant to add that fix to the main EBI Visual Framework as it could have other affects -- this issue of spacing should also be much better in v1.3 when we start using css' flexbox for the grid.

But,you should be able to add the snippet above to your project's css and sort the issue there -- if that works for you for now, I'll tag this issue to follow up on for v1.3.

**Comment by nbuso** [comment]
Thanks Ken, I will try the workaround.... and still have to update to 1.2

**Comment by khawkins98** [comment]
I've just committed a fix to hit in v1.3. If it doesn't resolve, or you need other help with the issue, please reopen.
